web/html/src/utils/test-utils/index.js

```diff
@@ -35,6 +40,20 @@ export const type = async (elementOrPromiseOfElement, text, options) => {
   return new Promise(resolve => window.requestAnimationFrame(() => resolve()));
 }
 
+/** Get a form field object */
+export const getFieldByName = (formName, fieldName) => {
+  return [...screen.getByTitle(formName).elements]
```

Comment:
Does the spread (`[...foo]`) solve a problem here over say `screen.getByTitle(formName).elements.filter(...)`? Right now it isn't very clear, unless I'm missing something here?

Reply:
This is because `screen.getByTitle(formName).elements` is not an Array, but an `HTMLFormControlsCollection`. Spreading it converts it into an Array (found the trick on the web).

Reply:
Sounds good, makes sense. 👍